### ISSUE TITLE
TypeError: Cannot call method 'split' of undefined

### ISSUE THREAD

**Issue by chook** [joshgummersall/statsd-alerting-backend#8]
I can't seem to get statsd-alerting-backend to work.

1. Start StatsD with statsd-alerting-backend using Slack dispatcher.
2. echo "test.localcounter:100|c" | nc -u -w 1 127.0.0.1 8125

Statsd log:

```
1 Sep 16:02:29 - DEBUG: Loading backend: statsd-alerting-backend

/Users/chenharel/Sparktale/statsd/node_modules/statsd-alerting-backend/lib/alert_distributor.js:101
      ref2 = data.split('|'), metric = ref2[0], type = ref2[1];
                  ^
TypeError: Cannot call method 'split' of undefined
    at AlertDistributor.module.exports.AlertDistributor.parsePacket (/Users/chenharel/Sparktale/statsd/node_modules/statsd-alerting-backend/lib/alert_distributor.js:101:19)
    at AlertDistributor.module.exports.AlertDistributor.onPacket (/Users/chenharel/Sparktale/statsd/node_modules/statsd-alerting-backend/lib/alert_distributor.js:128:16)
    at EventEmitter.<anonymous> (/Users/chenharel/Sparktale/statsd/node_modules/statsd-alerting-backend/lib/alert_distributor.js:2:57)
    at EventEmitter.emit (events.js:98:17)
    at Socket.<anonymous> (/Users/chenharel/Sparktale/statsd/node_modules/statsd/stats.js:172:21)
    at Socket.EventEmitter.emit (events.js:98:17)
    at UDP.onMessage (dgram.js:440:8)
```

**Comment by joshgummersall** [comment]
I will investigate this issue as soon as possible this week. The packet above seems valid so this shouldn't be an issue. I'll report back with what I find. Thanks for filing the issue!

**Comment by chook** [comment]
i investigated and will push a fix with PR later on.

**Comment by joshgummersall** [comment]
@chook awesome - thanks for doing that!

**Comment by joshgummersall** [comment]
Fixed in `0.1.5` release.
